source/contributed/remote-mining.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Remote mining
+contributed:
+    name: slowmotionghost
+    link: https://s[`Creep`](http://docs.screeps.com/api/#Creep)s.com/a/#!/profile/slowmotionghost
+    date: 2017-08-23
+---
+This article is a beginners guide to remote mining, and should provide ideas that can be worked towards, but leave the coding and implementation to you.
+
+## What is remote mining?
+Remote mining is the activity of harvesting [`Energy`](http://docs.screeps.com/api/#Resource) in rooms that are not owned by the player. To develop a productive economy, players should code a form remote mining to maximise their [`Energy`](http://docs.screeps.com/api/#Resource) income. There are many ways to perform remote mining, some of which will be discussed in this article.
+## Harvesting
+In order to obtain [`Energy`](http://docs.screeps.com/api/#Resource) from remote [`Sources`](http://docs.screeps.com/api/#Source), they need to be harvested, just like [`Sources`](http://docs.screeps.com/api/#Source) in owned rooms. [`Sources`](http://docs.screeps.com/api/#Source) in unclaimed rooms provides 1500 [`Energy`](http://docs.screeps.com/api/#Resource) per 300 ticks. If the room is reserved, this means the [`Source`](http://docs.screeps.com/api/#Source) provides 3000 [`Energy`](http://docs.screeps.com/api/#Resource) per 300 ticks. This shows the clear benefit of reserving a room, however remote harvesting can still be profitable without use of reservation. Each work part a [`Creep`](http://docs.screeps.com/api/#Creep) has can harvest 2 [`Energy`](http://docs.screeps.com/api/#Resource) per tick, meaning a [`Creep`](http://docs.screeps.com/api/#Creep) should have 5 work parts to fully harvest a [`Source`](http://docs.screeps.com/api/#Source) (in a reserved room). Depending on your collection method, a harvester may only require work and move parts, as [`Energy`](http://docs.screeps.com/api/#Resource) harvested by a [`Creep`](http://docs.screeps.com/api/#Creep) without a carry part is dropped and can be collected from the ground or a [`Container`](http://docs.screeps.com/api/#StructureContainer). It may be worth considering a carry part if your harvester [`Creep`](http://docs.screeps.com/api/#Creep) will be used to repair buildings in the remote room (such as a [`Container`](http://docs.screeps.com/api/#StructureContainer) it is using) or your harvester [`Creep`](http://docs.screeps.com/api/#Creep) will also be used to ferry the [`Energy`](http://docs.screeps.com/api/#Resource) back to your storage. The latter method is simple to implement as it requires only one role or job, however it is less [`Energy`](http://docs.screeps.com/api/#Resource) efficient to transport the work parts around, so having a dedicated harvester is worth considering.
```

Comment:
Might be worth to mention that the center 9 rooms of a sector contain 4000 energy in stead of the regular 1500/3000 combo. 

I would also reference the constants used (300 ticks = ENERGY_REGEN_TIME)
SOURCE_ENERGY_CAPACITY
SOURCE_ENERGY_NEUTRAL_CAPACITY
SOURCE_ENERGY_KEEPER_CAPACITY

This way the people implementing mining properly can use the constants of the game and avoid magic numbers 😄